From 416db33110f0392ba3f882f395168cbe37fed6ff Mon Sep 17 00:00:00 2001
From: "Levy, Lawrence-Jacques" <jacqlevy@cisco.com>
Date: Mon, 21 Sep 2015 18:43:39 +0200
Subject: [PATCH] Declare videonode dependency on Qt5MultimediaQuick_p

Change-Id: I896fda1689dba502380956c0b7a8a4e6b05a1e3a
Reviewed-by: Julien Brianceau <jbriance@cisco.com>
Reviewed-by: Oswald Buddenhagen <oswald.buddenhagen@theqtcompany.com>
Reviewed-by: Yoann Lopes <yoann.lopes@theqtcompany.com>

:Issues Addressed:
[PLAT-18000] Remove build and install for Qt5Widget

Upstream-Status: Cherry-picked from upstream
---
 src/src.pro |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/src.pro b/src/src.pro
index 26f99bc..e8d430e 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -20,6 +20,9 @@ qtHaveModule(quick) {
     src_imports.subdir = imports
     src_imports.depends = multimedia src_qtmultimediaquicktools
 
+    # For the videonode plugin
+    src_plugins.depends += src_qtmultimediaquicktools
+
     SUBDIRS += \
         src_qtmultimediaquicktools \
         src_imports
-- 
1.7.9.5

